docs/cvl/cvl2/changes.md

```diff
@@ -606,38 +649,38 @@ In CVL 1, you could write a sighash instead of a method identifier in the
 `methods` block.  This feature is no longer supported.  You will need to have
 the name and argument types of the called method in order to provide an entry.
 
-```{todo}
-If you do not change this, you will see the following error:
-```
+% ```{todo}
+% If you do not change this, you will see the following error:
+% ```
 
 ### `invoke`, `sinvoke`, and `call`
 
 Older versions of CVL had special syntax for calling contract and CVL functions:
- - `invoke f(args);` should be replaced with `f(args);`.
- - `sinvoke f(args);` should be replaced with `f@withrevert(args);`
+ - `invoke f(args);` should be replaced with `f@withrevert(args);`.
+ - `sinvoke f(args);` should be replaced with `f(args);`.
```

Comment:
take it or leave it "should be replaced with `f(args);` _or_ `f@norevert(args);`

Reply:
I'll leave it; I can't think of any reason to recommend `@norevert`.